File: _01_Languages/CSharp/src/_01_basicos/_06_EntradaSalida.cs
/*
 * 🧩 Enunciado:
 * Pide al usuario su nombre y su edad, y muestra un mensaje personalizado.
 */

using System;

class _06_EntradaSalida
{
    static void Main()
    {
        Console.Write("¿Cómo te llamas? ");
        string nombre = Console.ReadLine();

        Console.Write("¿Cuántos años tienes? ");
        int edad = Convert.ToInt32(Console.ReadLine());

        Console.WriteLine($"Hola {nombre}, tienes {edad} años. 👋");
    }
}

File: _01_Languages/Kotlin/src/_01_basicos/_06_EntradaSalida.kt
/*
 * 🧩 Enunciado:
 * Pide al usuario su nombre y edad, y muestra un mensaje personalizado.
 */

fun main() {
    print("¿Cómo te llamas? ")
    val nombre = readLine()!!

    print("¿Cuántos años tienes? ")
    val edad = readLine()!!.toInt()

    println("Hola $nombre, tienes $edad años. 👋")
}
